src/dbnode/persist/fs/msgpack/stream_with_digest_test.go

```diff
@@ -0,0 +1,143 @@
+package msgpack
```

Comment:
nit: Need the source file header.

Reply:
Whoops. Think I need to configure my IDE to add this automatically. Good catch